frontend/src/client/models.ts

```diff
@@ -33,6 +33,15 @@ export type ItemsPublic = {
   count: number
 }
 
+export type InquiryCreate = {
```

Comment:
I'd read the error text on this error on the Codacy site. I'm not sure this error isn't valid.